src/css/relay.css

```diff
@@ -242,12 +242,15 @@ button.fx-relay-modal-close-button {
 
 .fx-relay-menu-remaining-aliases {
   font-size: 14px;
-  padding: 0 24px;
-  color: var(--relayInk70);
+  width: 260px;
```

Comment:
Is the hard-width here required? It feels like a magic number and could break funny in different languages.